src/elastic/index.ts

```diff
@@ -22,7 +22,8 @@ export enum ContentCategory {
   NEWS = "news",
   UNCATEGORIZED = "uncategorized",
   CONTACT = "contact",
-  EXTERNAL = "service_external_link"
+  EXTERNAL = "service_external_link",
+  MANUAL = "manually_created_document"
```

Comment:
This is not correct. Manual creation should not affect the document category in any way, as at some point, we could have a need to manually create a document with specific category. That's why this should be its own boolean field in the document, probably something like "isManuallyCreated".

Reply:
Set as is_manually_created attribute.